### PR TITLE
MAT-7030: update cqm models

### PR DESCRIPTION
**Pull request by chubert-sb**
## MADiE PR Template

Jira Ticket: [MAT-7030](https://jira.cms.gov/browse/MAT-7030)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e. Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/models/cqm/datacriteria/EncounterPerformed.java [hunk 1]
```diff
@@ -35,4 +35,5 @@ public class EncounterPerformed extends DataElement {
   private String qdmCategory = "encounter";
   private String qdmStatus = "performed";
   private String qdmVersion = "5.6";
+  private String _type = "QDM::EncounterPerformed";
```
Comment (adongare) [inline src/main/java/gov/cms/madie/models/cqm/datacriteria/EncounterPerformed.java:39]: I see one issue in this class related to `private Code clazz`. Client is going to send `Code class` but because the class is reserved word in java we renamed it to `clazz`. we need to have a way to convert this back and forth otherwise we are going to lose it.

Comment (adongare) [reply src/main/java/gov/cms/madie/models/cqm/datacriteria/EncounterPerformed.java:39]: new task is fine too but we need to note this so that we don't forget. I don't remember if there was any other such scenario.

**Review by adongare — DISMISSED** [review]

File: src/main/java/gov/cms/madie/models/cqm/datacriteria/EncounterPerformed.java (hunk 1 above)
Comment (chubert-sb) [reply src/main/java/gov/cms/madie/models/cqm/datacriteria/EncounterPerformed.java:39]: Ah I see what you mean, I originally modeled these after the ruby representations which does have `clazz` but the javascript does have `class` and a little conversion method for it. That's all a good point. I'll add a note in here in case we need something like that.

**Review by adongare — APPROVED** [review]
